crates/ty/src/args.rs

```diff
@@ -51,21 +51,14 @@ pub(crate) struct CheckCommand {
     #[arg(long, value_name = "PROJECT")]
     pub(crate) project: Option<SystemPathBuf>,
 
-    /// Path to the Python environment.
+    /// Path to your project's Python environment or interpreter.
     ///
-    /// ty uses the Python environment to resolve type information and third-party dependencies.
+    /// Ty uses your Python environment to resolve third-party imports in your code.
```

Comment:
```suggestion
    /// ty uses your Python environment to resolve third-party imports in your code.
```

I believe using Ty is incorrect according to our branding guide